maas/meta/backupninja.yml

```diff
@@ -0,0 +1,6 @@
+backup:
+  maas:
+    fs_includes: []
+    fs_excludes: []
+    actions:
+    - cmd: "rsync -aH  --exclude='/var/lib/maas/boot-resources' --relative /etc/maas /var/lib/maas /var/backups/maas_config"
```

Comment:
if I use fs_include/exclude configs will be sent only to remote location, but idea was also to store them on glusterfs volume. in other words having local backup that is quickly accessible but not stored on the same node that is being backed up and remote copy